### PR TITLE
Remove old/unused code

### PR DESCRIPTION
**Pull request by ariessunfeld**
- The first commit should be simple to merge
- In a second commit, I removed a lot of functions from `helpers.py` and their associated tests, since they were not in use in `CraterDatabase`. Unsure if you want to take all these changes.
- Started trying to increase coverage in third commit, which may be a little out of scope of this PR

### CODE REVIEW

File: craterpy/helper.py [hunk 1]
```diff
@@ -1,86 +1,12 @@
...
-import numpy as np
-import pandas as pd
-
 
 # Geospatial helpers
-def lon360(lon):
-    """Return longitude in range [0, 360)."""
-    return (lon + 360) % 360
-
-
```
Comment (cjtu) [inline craterpy/helper.py:12]: This will be useful later I think.

File: craterpy/helper.py [hunk 2]
```diff
@@ -1,86 +1,12 @@
...
-    False
-    >>> inglobal(0, -50)
-    True
-    """
-    return (-90 <= lat <= 90) and (0 <= lon360(lon) <= 360)
-
-
-def get_spheroid_rad_from_wkt(wkt):
-    """Return body radius from Well-Known Text coordinate reference system."""
-    return float(wkt.lower().split("spheroid")[1].split(",")[1])
```
Comment (cjtu) [inline craterpy/helper.py:81]: let's keep this one for now too

File: craterpy/tests/test_classes.py
Comment (cjtu) [inline craterpy/tests/test_classes.py:1]: i would leave these out of this pr. also, it's lot of logic to test one parameter in each case. we should think about the tech debt / maintenance overhead of introducing 100 lines of code to test 4 lines

File: craterpy/tests/test_helper.py [hunk 3]
```diff
@@ -10,30 +10,6 @@
...
-        actual = ch.lon360(355)
-        expected = 355
-        self.assertEqual(actual, expected)
-        actual = ch.lon360(-175)
-        expected = 185
-        self.assertEqual(actual, expected)
-        actual = ch.lon360(-5)
-        expected = 355
-        self.assertEqual(actual, expected)
-
```
Comment (cjtu) [inline craterpy/tests/test_helper.py:36]: lol this test is really funny to me in hindsight. good example of overtesting. We just need to see that the edge cases and different types of inputs work. here we only test int and it's unclear what e.g. 20 reveals that other inputs wouldn't already show

**Review by cjtu — CHANGES_REQUESTED** [review]
Looks pretty good. If you remove that test coverage commit and add back in the couple helpers, i can merge. You can choose commits to keep or drop with the `git rebase -i HEAD~4` then `pick` or `drop` individual commits ([guide](https://dev.to/1407arjun/how-to-use-git-rebase-to-alter-previous-commits-in-the-project-history-3120)).
